numpy/core/fromnumeric.py

```diff
@@ -3558,6 +3558,10 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
     where : array_like of bool, optional
         Elements to include in the standard deviation.
         See `~numpy.ufunc.reduce` for details.
+    mean : optionally provide the mean to prevent recalculation, the mean
+          should have a shape as if it was calculated with keepdims = True
+          and the axis the same as used in the call to this std function. 
+          For masked arrays this argument is ignored.
 
         .. versionadded:: 1.20.0
```

Comment:
This `versionadded` belongs to `where` - you should add a new one with `1.26.0`.

Reply:
Corrected locally now.